static/src/javascripts/projects/commercial/adblock-ask.ts

```diff
@@ -23,16 +22,12 @@ const canShow = () =>
 export const initAdblockAsk = (): Promise<void> => {
 	if (!canShow()) return Promise.resolve();
 
-	return (
-		fastdom
-			// eslint-disable-next-line @typescript-eslint/no-unsafe-return -- TODO: fix it
-			.measure(() => $('.js-aside-slot-container'))
-			.then((slot) => {
-				if (!slot) return;
-				return fastdom.mutate(() => {
-					// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call -- TODO: fix it
-					slot.append(askHtml);
-				});
-			})
-	);
+	return fastdom
+		.measure(() => document.querySelector('.js-aside-slot-container'))
+		.then((slot) => {
+			if (!slot) return;
+			return fastdom.mutate(() => {
+				slot.insertAdjacentHTML('beforeend', askHtml);
```

Comment:
This maintains the original JS logic.

Reply:
Could we add a test to capture the insertion of this element into the dom and the generated URL?

Reply:
I was thinking about this and just got a tad lazy :)